src/components/plots/xyPlot.ts

```diff
@@ -158,69 +201,26 @@ module Plottable {
      * This call does not override auto domain adjustment behavior over visible points.
      */
     public showAllData() {
+      this._updateExtentsForAttr("x");
       this._xScale.autoDomain();
-      if (!this._autoAdjustYScaleDomain) {
-        this._yScale.autoDomain();
-      }
+      this._updateExtentsForAttr("y");
+      this._yScale.autoDomain();
+      return this;
     }
```

Comment:
I remember there used to be some confusion on the point of this endpoint.  If this is useful to users, then keep it.  Otherwise don't.